src/wasm.h

```diff
@@ -583,15 +583,15 @@ class Literal {
   }
   Literal rotL(const Literal& other) const {
     switch (type) {
-      case WasmType::i32: return Literal(RotateLeft(uint32_t(i32), uint32_t(other.i32)));
-      case WasmType::i64: return Literal(RotateLeft(uint64_t(i64), uint64_t(other.i64)));
+      case WasmType::i32: return Literal(RotateLeft(uint32_t(i32), other.i32));
```

Comment:
`RotateLeft` is a template where the 2 arguments are the same type. How does changing this from an explicit cast to an implicit conversion help?